src/summersplash2022/utils/handBookPages.tsx

```diff
@@ -0,0 +1,130 @@
+import { WhichButtonPressed } from './utils';
+import style from '../index.module.css';
+
+const HandBooKPages = ({ selectedButton }: WhichButtonPressed) => {
+  if (selectedButton === 'Formmal og verdier') {
+    return (
+      <div>
+        <h3 className={style.handbookHeaderDeskotop}>Formmål og verdier</h3>
+        <p className={style.positionTextSubLeft}>
+          Så, hvorfor Variant? Hvorfor er vi egentlig til? Vårt formål er å
+          utvikle samfunnet vi lever i. Flinke personer som tenker nye tanker,
+          og så lager de riktige løsningene.
+          <strong>
+            <br></br>
```

Comment:
Når man kjenner behovet for en `break-line` er det ofte et tegn på at her burde vi introdusere en klasse med litt styling. Break-lines (`<br />`) er ment for å bryte opp en setning/avsnitt rent typografisk og ikke til å skape luft fra estetiske grunner. Så her mistenker jeg at det egentlig er tenkt en `h4` som har noe `margin-top`. Da blir det og mer semantisk tydelig og ting som skjermlesere og outline generatorer får mer tydelig viktighet i overskriftene (viktig for SEO også).